tests/smoke/test_ucc_build.py

```diff
@@ -294,3 +295,62 @@ def test_ucc_generate_with_configuration_files_only():
         for f in files_to_not_exist:
             expected_file_path = path.join(expected_folder, *f)
             assert not path.exists(expected_file_path)
+
+def test_ucc_generate_openapi_with_configuration_files_only():
+    with tempfile.TemporaryDirectory() as temp_dir:
+        package_folder = path.join(
+            path.dirname(path.realpath(__file__)),
+            "..",
+            "testdata",
+            "test_addons",
+            "package_no_global_config",
+            "package",
+        )
+        ucc.generate(source=package_folder, outputdir=temp_dir, openapi=True)
+
+        openapi_file = Path(Path(temp_dir) / "Splunk_TA_UCCExample/static/openapi.json")
+        assert not openapi_file.exists()
+        
+def test_ucc_generate_openapi_with_configuration():
```

Comment:
I think you can just extend the existing test `test_ucc_generate_with_inputs_configuration_alerts`.

Reply:
approach I was using so far was not to touch existing unit tests and just make sure they are satisfied